res/lang/rooms/rooms_en.txt

```diff
@@ -1,44 +1,44 @@
-ROOM_ELEVATORPLAN        = Elevator target planner
-ROOM_NEWSPLANNER         = News planner
+ROOM_ELEVATORPLAN        = Elevator Room Planner
```

Comment:
Dunno if there is an even better name for it.

In contrast to the real "Room Plan" you cannot switch signs here. It is only useful to "select a target room to move to". As if next to the elevator doors you already had an array of buttons to directly choose the target, not just "up / down" buttons to call the elevator.

Reply:
Elevator target sounded like you were just selecting the floor the elevator should go to, but you're selecting the floor and room, so I felt that elevator room planner was a good compromise.

Reply:
How about just "Elevator"?

Reply:
Could do that but my vote is on the translation I gave, it's more descriptive and tells you you'll be able to get to the various rooms through the elevator.

Reply:
For me, your proposal is fine.